### ISSUE TITLE
Migrate emojis

### ISSUE THREAD

**Issue by daita** [nextcloud/social#1039]
Twemoji is stuck to 0.12.0, maybe using another component ? 

What about the one used in Talk ?

**Comment by violoncelloCH** [comment]
yes, we should use the (meanwhile) existing emoji picker component: https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/EmojiPicker/EmojiPicker.vue

**Comment by violoncelloCH** [comment]
implementing this possibly also fixes  #111

**Comment by SchBenedikt** [comment]
Automatically closed — the emoji migration has been completed as part of the broader app rewrite. No separate emoji migration file is needed; emojis are handled via the ActivityPub output.
